pex/pep425tags.py

```diff
@@ -12,345 +15,317 @@
 from __future__ import absolute_import
 
 import distutils.util
-import logging
+from distutils import log
```

Comment:
This appears diverged from original: https://github.com/pypa/pip/blob/8f4f15a5a95d7d5b511ceaee9ed261176c181970/src/pip/_internal/pep425tags.py#L5.

Reply:
That may be so. As per the header comment I copied from our vendored setuptools.